README.md

```diff
@@ -127,52 +136,59 @@ For more information, click [here](https://docs.aws.amazon.com/cognito/latest/de
 
 **Retrieving an ID token using aws.cognito.signin.user.admin**
 
-To access the FHIR API, an ID token is required. A Cognito ID token can be obtained using the following command substituting all variables with their values from `INFO_OUTPUT.log` or by using the `serverless info --verbose` command.
-+	For Windows, enter:
+To access the FHIR API, an ID token is required. A Cognito ID token can be obtained using the following command, substituting all variables with values from `INFO_OUTPUT.log` or by using the `serverless info --verbose` command.
++	For Windows, you will need Python 3 installed. Enter:
```

Comment:
Can we state Python 3.0 instead of Python 3? (Reference: https://www.python.org/download/releases/3.0/)